From 8220387ce2fa8e261c93a0a95c9771b4af34c9be Mon Sep 17 00:00:00 2001
From: Brecht Van Lommel <brecht@blender.org>
Date: Thu, 12 Jun 2025 14:43:47 +0200
Subject: [PATCH 15/22] BLENDER: Fine tune pages considered expensive

---
 routers/common/blockexpensive.go | 25 +++++++++++++------------
 1 file changed, 13 insertions(+), 12 deletions(-)

diff --git a/routers/common/blockexpensive.go b/routers/common/blockexpensive.go
index fec364351c..f9d05836cd 100644
--- a/routers/common/blockexpensive.go
+++ b/routers/common/blockexpensive.go
@@ -40,25 +40,26 @@ func isRoutePathExpensive(routePattern string) bool {
 
 	expensivePaths := []string{
 		// code related
-		"/{username}/{reponame}/archive/",
+		// "/{username}/{reponame}/archive/",
 		"/{username}/{reponame}/blame/",
-		"/{username}/{reponame}/commit/",
-		"/{username}/{reponame}/commits/",
+		// "/{username}/{reponame}/commit/",
+		// "/{username}/{reponame}/commits/",
 		"/{username}/{reponame}/compare/",
-		"/{username}/{reponame}/graph",
-		"/{username}/{reponame}/media/",
-		"/{username}/{reponame}/raw/",
+		// "/{username}/{reponame}/graph",
+		// "/{username}/{reponame}/media/",
+		// "/{username}/{reponame}/raw/",
 		"/{username}/{reponame}/rss/branch/",
-		"/{username}/{reponame}/src/",
+		// "/{username}/{reponame}/src/",
+		"/{username}/{reponame}/compare/",
 
 		// issue & PR related (no trailing slash)
-		"/{username}/{reponame}/issues",
-		"/{username}/{reponame}/{type:issues}",
-		"/{username}/{reponame}/pulls",
-		"/{username}/{reponame}/{type:pulls}",
+		// "/{username}/{reponame}/issues",
+		// "/{username}/{reponame}/{type:issues}",
+		// "/{username}/{reponame}/pulls",
+		// "/{username}/{reponame}/{type:pulls}",
 
 		// wiki
-		"/{username}/{reponame}/wiki/",
+		// "/{username}/{reponame}/wiki/",
 
 		// activity
 		"/{username}/{reponame}/activity/",
-- 
2.47.3

