From f3e7409d4431c9acdc578617051c10874d718fa8 Mon Sep 17 00:00:00 2001
From: Bart van der Braak <bart@blender.org>
Date: Tue, 7 Oct 2025 15:02:05 +0200
Subject: [PATCH 19/19] BLENDER: Add Git LFS for Git Hook scripts

---
 Dockerfile.rootless | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Dockerfile.rootless b/Dockerfile.rootless
index ebca19ca5b..8444b2fd35 100644
--- a/Dockerfile.rootless
+++ b/Dockerfile.rootless
@@ -50,6 +50,7 @@ RUN apk --no-cache add \
     dumb-init \
     gettext \
     git \
+    git-lfs \
     curl \
     gnupg \
     openssh-keygen \
-- 
2.49.1

